packages/espressocash_app/lib/features/outgoing_split_key_payments/models/outgoing_split_key_payment.dart

```diff
@@ -47,17 +49,20 @@ class OSKPStatus with _$OSKPStatus {
     required Uri link2,
     Uri? qrLink,
     required EscrowPrivateKey escrow,
+    DateTime? timestamp,
```

Comment:
Here, vice versa, you don't need a timestamp on the status, you already have a field on the `OutgoingSplitKeyPayment` model.